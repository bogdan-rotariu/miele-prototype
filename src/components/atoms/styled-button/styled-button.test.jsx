import { render } from '@testing-library/react'
import { Button } from '.'

describe('Button', () => {
    it('should match snapshot', () => {
        const { container } = render(
            <Button onClick={jest.fn()}>button</Button>,
        )
        expect(container.firstChild).toMatchSnapshot()
    })

    it('should match snapshot if disabled', () => {
        const { container } = render(
            <Button onClick={jest.fn()} isDisabled>
                button
            </Button>,
        )
        expect(container.firstChild).toMatchSnapshot()
    })
})
