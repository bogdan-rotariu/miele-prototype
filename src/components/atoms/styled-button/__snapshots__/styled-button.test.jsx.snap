// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot 1`] = `
<button
  class="button"
>
  button
</button>
`;

exports[`Button should match snapshot if disabled 1`] = `
<button
  class="button disabled"
  disabled=""
>
  button
</button>
`;
